src/topology/uniform_space/basic.lean

```diff
@@ -1191,6 +1191,25 @@ begin
   exact h.continuous
 end
 
+@[to_additive]
+instance [uniform_space α] : uniform_space (αᵐᵒᵖ) :=
+uniform_space.comap mul_opposite.unop ‹_›
+
+@[to_additive]
+lemma uniformity_mul_opposite [uniform_space α] :
+  𝓤 (αᵐᵒᵖ) = comap (λ q : αᵐᵒᵖ × αᵐᵒᵖ, (q.1.unop, q.2.unop)) (𝓤 α) :=
+rfl
+
+@[to_additive]
+lemma uniform_continuous_mul_unop [uniform_space α] :
```

Comment:
Or `mul_opposite.uniform_continuous_unop`. Then you can open one of the namespaces `mul/add_opposite` and use just `uniform_continuous_unop`. Also, it agrees with `add/mul_opposite.continuous_op/unop`.

Reply:
Done